src/auth/token.tsx

```diff
@@ -31,7 +31,7 @@ export interface TokenService {
   readonly getRefreshToken: () => string | null;
   readonly setRefreshToken: (token: string) => void;
   readonly removeRefreshToken: () => void;
-  readonly getPrivilegeLevel: () => number;
+  readonly getPrivilegeLevel: () => number | string;
```

Comment:
We should keep this as all number or all string, the enum above can be updated so that NONE = 'none'.

Reply:
Updated `getPrivilegeLevel()` too